composer/trainer/trainer_hparams.py

```diff
@@ -453,6 +455,10 @@ def validate(self):
         if self.scale_schedule_ratio <= 0:
             raise ValueError("scale_schedule_ratio must be a positive value.")
 
+        if isinstance(self.grad_accum, str) and self.grad_accum != "auto" or isinstance(self.grad_accum,
+                                                                                        int) and self.grad_accum < 1:
+            raise ValueError("grad_accum must be ``auto`` or an int greater than or equal to 1")
```

Comment:
```suggestion
            raise ValueError('grad_accum must be "auto" or an int greater than or equal to 1')
```

Reply:
adjusted. What's the stylistic rule here for ' vs " vs. `? I've seen in other parts """ + `` so that's what I tried emulating

Reply:
The one common rule is that if your string itself contains double quotes or single quotes, you should use the opposite type of quotes to encapsulate the string.

Reply:
Here, I wanted you to use quotes to surround "auto" just to indicate in the printed message that it represents a string.

Reply:
Got it, makes sense! Thanks for the explanation